themes/digital.gov/src/scss/new/_uswds-overrides.scss

```diff
@@ -2,4 +2,8 @@
 
 .usa-identifier img[alt] {
   @include u-text('white');
+}
+
+.usa-in-page-nav__item--sub-item {
+  @include u-display('none')
```

Comment:
Let's run prettier on this file to get consistent double quotes.

Reply:
Done and done